### DISCUSSION TITLE
Add a configuration option for displaying disturbance messages instantly

### DISCUSSION THREAD

**Discussion by Option212** [yyrkoon94/lovelace-idf-mobilite#48] [Ideas]
Bonjour,

Merci pour votre travail !

Pourriez-vous ajouter une option pour afficher le message de perturbation en instantanée (format de votre choix) afin d'éviter de l'avoir en défilé et difficilement lisible ?

****
Hello,

Thank you for your  work!

Could you please add an option to display the disruption message immediately (in your preferred format) so that it remains static and easily readable, rather than scrolling?

**Comment by yyrkoon94** [comment]
Hi @Option212,

Merci ;) Tu vois ça comment ? Une carte dédiée aux message que tu afficherais à côté de la carte des horaires ?

**Comment by Option212** [comment]
Bonjour @yyrkoon94,

Plusieurs idées : 

1. Un système de couleur avec ligne X : <bouton de couleur> 
- Vert : RAS  ;
- Orange (perturbation moyenne) ;
- Rouge (forte perturbation). 
Quand on clique sur les boutons Orange & Rouge on a l'ensemble de l'information qui nous est renvoyé dans le pop d'HomeAssistant.

Example sous-jacent fait avec [mushrooms](custom:mushroom-entity-card).
![image](https://github.com/user-attachments/assets/242e38c8-761a-489c-ae4f-5a670a8e3049)

2. Ajouter un onglet dans le tableau de bord : 
- Onglet 1 affichage classique RATP ;
- Onglet 2, texte entier concernant la perturbation.

**Comment by yyrkoon94** [comment]
Hello @Option212,

En attendant d'être mieux inspiré, j'ai ajouté une option pour afficher le message en entier au lieu de le faire défier (v0.3.0)
